File: dev/requirements/screen/cases/pacing-bar.2.9.case.swift
#if canImport(SwiftUI) && os(iOS)
import SetlistUI

/// 2.9 — the set's shape in one strip — three told, one live, five queued, each sized by its estimate.
let pacingBar_2_9 = RequirementCase(id: "2.9") {
    try MainActor.assumeIsolated {
        try expectScreen(StageScreen(state: StageFixture.state()), slug: "pacing-bar", id: "2.9")
    }
}
#endif

File: dev/requirements/screen/cases/pacing-bar.2.9.case.dart
import 'package:setlist_ui/setlist_ui.dart';

import '../../shared/requirement_case.dart';
import '../../shared/snapshot.dart';

/// 2.9 — The pacing bar carries one segment per joke in set order, sized by estimated
/// length, coloured told, live or queued.
final pacingBar_2_9 = RequirementCase('2.9', (tester) async {
  await expectRegion(
    tester,
    StageFixture.state(),
    region: StageRegion.pacingBar,
    slug: 'pacing-bar',
    id: '2.9',
  );
});
